cmd/system-probe/modules/eventmonitor_linux.go

```diff
@@ -25,6 +27,10 @@ var EventMonitor = module.Factory{
 	},
 }
 
-func createProcessMonitorConsumer(evm *eventmonitor.EventMonitor) (eventmonitor.EventConsumerInterface, error) {
+func createProcessMonitorConsumer(evm *eventmonitor.EventMonitor, config *netconfig.Config) (eventmonitor.EventConsumerInterface, error) {
+	if !usmconfig.IsUSMSupportedAndEnabled(config) || !usmconfig.NeedProcessMonitor(config) {
+		return nil, nil
+	}
+
 	return procmon.NewProcessMonitorEventConsumer(evm)
 }
```

Comment:
Is it only for USM?

Reply:
Yes, this particular consumer is only used for USM at the moment.